docs/atlascli/command/atlas-backups-exports-buckets-list.txt

```diff
@@ -0,0 +1,79 @@
+.. _atlas-backups-exports-buckets-list:
+
+==================================
+atlas backups exports buckets list
+==================================
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+List cloud backup restore buckets for your project and cluster.
+
+Syntax
+------
+
+.. code-block::
+
+   atlas backups exports buckets list [options]
+
+Options
+-------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -h, --help
+     - 
+     - false
+     - help for list
+   * - --limit
+     - int
+     - false
+     - Number of items per page. This value defaults to 100.
+   * - -o, --output
+     - string
+     - false
+     - Output format. Valid values are json, json-path, go-template, or go-template-file.
+   * - --page
+     - int
+     - false
+     - Page number. This value defaults to 1.
+   * - --projectId
+     - string
+     - false
+     - Project ID to use. Overrides the settings in the configuration file or environment variable.
+
+Inherited Options
+-----------------
+
+.. list-table::
+   :header-rows: 1
+   :widths: 20 10 10 60
+
+   * - Name
+     - Type
+     - Required
+     - Description
+   * - -P, --profile
+     - string
+     - false
+     - Profile to use from your configuration file.
+
+Examples
+--------
+
+.. code-block::
+
+   The following example retrieves the continuous backup export buckets for the cluster Cluster0:
```

Comment:
**Question**: I don't see Cluster0 specified in this command, is this accurate? Just verifying it should be in this description

Reply:
@sarahsimpers You're correct, this command doesn't take the name of the cluster as a parameter. I fixed that.